templates/repo/scripts/component.mk

```diff
@@ -102,7 +103,7 @@ endif
 .PHONY: init
 
 check-plan: init check-auth ## run a terraform plan and check that it does not fail
-	@$(terraform_command) plan $(TF_ARGS) -detailed-exitcode -lock=false; \
+	@$(terraform_command) plan $(TF_ARGS) -detailed-exitcode -lock=false -out=$(CHECK_PLANFILE_PATH) ; \
```

Comment:
Do you think we should still output the human-readable plan?

Reply:
i think tf show <planfile> would work for that

Reply:
It looks like it still does it